src/files/FilesListener.h

```diff
@@ -25,7 +25,7 @@ class FilesListener : public executor::ExecuteEventListener {
 private:
     const bool suppressStderr_;
     std::vector<int> fds_;
-    int devnull_;
+    int devnull_{};
```

Comment:
co? przy intach też piszemy puste inicjalizatory?
IMO to nie pomaga i wygląda trochę śmiesznie, już czytelniej by było ` = 0;`

Reply:
Można by się spierać, która z wersji jest lepsza (`{}` działa z każdym typem, itd). Tutaj zmienię na `-1` (tak jak było w nieopublikowanej wersji którą robiłem ręcznie).

Reply:
OK, tu może i zmień na `-1`, bo to FDk, ale jest cała masa innych miejsc w których masz taki inicjalizator, i IMO to jest niepotrzebne. Implicitna inicjalizacja jest ok, szczególnie przy structach które potem i tak wypełniasz. A przy intach to jest IMO bardzo nieintuicyjne,